@lingosta/app/components/Layout/Navbar.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+
+import React from 'react'
+import useUser from '../../hooks/useUser'
+import Link from 'next/link';
+
+const Navbar = () => {
+
+    const { user, logout } = useUser();
+
+    return (
+        <div className='border-b-[1px] border-neutral-200 p-5'>
+            <div className='flex justify-between'>
+                <Link href='/' className='text-lg font-semibold'>
+                    Lingosta
+                </Link>
+                <div>
+                    { !user && (
```

Comment:
No need to change but just an FYI that you can also do `!user ? (<loggedInStuff/>) : (<notLoggedInStuff />)`